k8s/safe/Identity.yaml

```diff
@@ -0,0 +1,27 @@
+# /*
```

Comment:
I had to revert these from the archive folder temporarily.

Once @abhishek44sharma fixes the `make`-related issues, we can move them to archive again and use the generated files instead.